index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+     <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+
+     </head>
+     <body>
+       <header>
+         <div class="head-wrapper">
+           <nav class="nav-bar">
+             
+              <a class="logo"  href="#">My Logo</a>
+            
+             <div class="menu">
+                <div class="toogle-btn"></div>
+                <div class="toogle-btn"></div>
+                <div class="toogle-btn"></div>
+             </div>
+
+           </nav>
+         </div>
+       </header>
+
+       <section>
+        <div class="headline-wrapper">
+           <div class="headline-container">
+             <div class="intro"><h1>I'm Vana<br> Glad to see you!</h1></div>
```

Comment:
- [ ] [OPTIONAL] Hi, well done with your design 👏. thank you for implementing the required changes from previous reviewers. Something you can add to your project is to ensure that your `font-weight` also matches the Figma. It's important to practice having a keen eye when looking at the design, in this case, if you look at the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) you will notice that the font-weight for the `h1` should be 700.
something like this 
```css
font-weight: 700;
```